packages/twitch-pubsub-client/src/Messages/PubSubUndocumentedMessage.ts

```diff
@@ -0,0 +1,33 @@
+import { Enumerable } from '@d-fischer/shared-utils';
+import { rtfm } from 'twitch-common';
+
+/** @private */
+export interface PubSubUndocumentedMessageContent {
+	data: unknown;
+}
+
+/** @private */
+export interface PubSubUndocumentedMessageData {
+	data: PubSubUndocumentedMessageContent;
+	type: string;
+}
+
+/**
+ * A message that informs about a user unlocking a new bits badge.
+ */
+@rtfm<PubSubUndocumentedMessage>('twitch-pubsub-client', 'PubSubUndocumentedMessage', 'data')
```

Comment:
The third parameter is for specifying an identifier to output. Since unknown data has no way to determine an identifier, please just remove the third parameter as well as the type parameter.